content/es/shift-left.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Shift Left
+status: Completed
+category: Concept
+tags: ["methodology", "", ""]
+---
+
+## Lo que es
+
+Izquierda en Shift Left se refiere a etapas anteriores en un ciclo de vida de desarrollo de software,
+pensando en el ciclo de vida como una línea donde las etapas se ejecutan de izquierda a derecha.
+Shift Left es la práctica de implementar pruebas, seguridad u otras prácticas de desarrollo
+temprano en el ciclo de vida de desarrollo de software en lugar de hacia el final.
+
+Aunque originalmente se usaba para referirse al proceso de prueba temprana,
+Shift Left ahora también se puede aplicar a otros aspectos del desarrollo de software y [DevOps](/es/devops/), como la seguridad y la implementación.
+
+## Problema que aborda
+
+Los problemas de seguridad, los errores y los defectos de software pueden ser más difíciles y costosos de solucionar.
+si se descubren tarde en el ciclo de desarrollo o después de la implementación,
+particularmente si el software ya se ha implementado en producción.
+
+## Cómo ayuda
+
+Al adoptar una mentalidad de cambio a la izquierda para el desarrollo de software,
+los equipos pueden implementar pruebas y seguridad a lo largo del ciclo de vida del desarrollo.
+Y porque la responsabilidad de las pruebas y la seguridad se comparte en todo el equipo de desarrollo.
```

Comment:
I think this could be a better translation:

```suggestion
Debido a que la responsabilidad de las pruebas y la seguridad se comparte en todo el equipo de desarrollo.
```